defaults/main.yml

```diff
@@ -552,7 +548,7 @@ maas_pip_packages:
   - python-keystoneclient
   - python-magnumclient
   - python-neutronclient
-  - python-novaclient<7.0.0
+  - python-novaclient
```

Comment:
@prometheanfire Awesome! Please give it an approved review if everything looks good to you.